core/tests/protractor_utils/CollectionEditorPage.js

```diff
@@ -135,32 +140,36 @@ var CollectionEditorPage = function() {
   this.publishCollection = async function() {
     await waitFor.elementToBeClickable(
       editorPublishButton, 'Collection Publish button is not clickable');
-    await editorPublishButton.click();
+    await action.click('Editor Publish Button', editorPublishButton);
   };
 
   // Set collection title.
   this.setTitle = async function(title) {
-    await editorTitleInput.sendKeys(title);
+    await action.sendKeys('Editor Title Input', editorTitleInput, title);
   };
 
   // Set collection objective.
   this.setObjective = async function(objective) {
-    await collectionEditorObjectiveInput.sendKeys(objective);
+    await action.sendKeys(
+      'Collection Editor Objective Input',
+      collectionEditorObjectiveInput, objective);
   };
 
   // Set collection category.
   this.setCategory = async function(category) {
-    await editorCategoryDropdown.first().click();
-    await browser.driver.switchTo().activeElement().sendKeys(category + '\n');
+    await action.click(
+      'Editor Category Drop Down', editorCategoryDropdown.first());
+    var test = await browser.driver.switchTo().activeElement();
```

Comment:
Can we call the variable "categoryItemElement" instead?

Reply:
@kevintab95 actually I'm stuck on the same error from the past few days ,the one you requested a change for. It is throwing 
`Failed: Cannot read property 'bind' of undefined` at line `163:18`.I am not sure how to resolve it

Reply:
Oh I see. In that case, let's revert back to how it was earlier. You can add a comment in the code explaining why this wasn't changed. Thanks!

Reply:
@kevintab95 Done !.Circle-Ci tests are giving some weird error ,I think it is a flake .Also some e2e_group_ tests are added ,what are these?

Reply:
@mridul-netizen can you please merge from develop per this [thread](https://groups.google.com/g/oppia-dev/c/Z-t33KYlenc)?